extensions/sql-database-projects/src/models/project.ts

```diff
@@ -614,18 +688,25 @@ export class Project {
 /**
  * Represents an entry in a project file
  */
-export class ProjectEntry {
+export abstract class ProjectEntry {
```

Comment:
The change that caused a lot of line changes in this PR was making ProjectEntry and abstract class because SQLCMD variable project entries don't have fsUri and relativePath